tools/diff/diff.xml

```diff
@@ -12,40 +13,93 @@
     </stdio>
 
     <command ><![CDATA[
-        diff -u '$input1' '$input2' > '$output1'
+        diff -u '$input1' '$input2' > '$diff_file';
+        #if $generate_report:
+            sed -e 's/@@title@@/Diff report for $input1.name and $input2.name/g'
```

Comment:
$input1.name --> use $input.element_identifier this should also work for collections. And please put them into single quotes.

Reply:
Done! Thanks for the comments!
If you see anything else or have some more usability ideas, please let me know :)